test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1542,14 +1541,15 @@ public void testGetDocContentSCVerifyGroupPush() throws Exception {
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
     GetContentsRequest request = new GetContentsRequest(
         new DocId("http://localhost:1/sites/SiteCollection"));
-    GetContentsResponse response = new GetContentsResponse(baos);
-    // First call returns 3 groups(Owners, Members with 2 users and Visitors)
-    adaptor.getDocContent(request, response);
+    GetContentsResponse response1 = new GetContentsResponse(baos);
+    GetContentsResponse response2 = new GetContentsResponse(baos);
```

Comment:
The pattern is to get this just before each getDocContent request, but we can do that in the followup CL to replace GetContentsResponse with RecordingResponse. The latter also has an implicit input stream, which is especially handy when you don't care about the content.

Reply:
ok. Thank you.